_platforms/en/datigov.md

```diff
@@ -29,9 +30,9 @@ Our aim is to evolve the current online portal of open data to a system that pro
 
 As with all the projects of Developers Italia, the portal of open data follows a development model based on a collaborative process  which is typical of the community of developers. The tools that we want to use to structure the development process are:
 
-  * A development roadmap agile and public 
+  * A development roadmap agile and public
```

Comment:
```suggestion
  * A public development roadmap
```

Reply:
Thanks @libremente ! I'll put this fix inside the PR but I'd like open specific PRs for contents review